packages/code-snippet/style/index.css

```diff
@@ -79,6 +83,18 @@
   padding: 0;
 }
 
+.elyra-codeSnippet-item textarea {
+  background-color: inherit;
+  border: none;
+  font-family: var(--jp-code-font-family);
+  font-size: var(--jp-code-font-size);
+  line-height: var(--jp-code-line-height);
+  min-height: 125px;
+  height: 100%;
+  width: 100%;
+  resize: vertical;
```

Comment:
I believe you need a few more lines so it does horizontal scrolling instead of word wrapping 
```suggestion
  resize: vertical;
  white-space: pre;
  overflow: auto;
  overflow-wrap: normal;
```
I got this from https://stackoverflow.com/a/13446005 and tested it locally

Reply:
updated https://github.com/elyra-ai/elyra/pull/431/commits/8ade4906d473ec06a006ef57e32a41f13e38c1f6

Reply:
that doesn't fully fix the issue. It doesn't wrap by default, only once you resize the textarea

Reply:
i am not sure what you mean. should it wrap or scroll? i initially had it wrapping but change change to scroll after initial review comment:

![image](https://user-images.githubusercontent.com/13156555/79368569-9c394700-7f1d-11ea-8017-c3925f7bfc84.png)

Reply:
this is what mine looks like:
<img width="1792" alt="Screen Shot 2020-04-15 at 10 11 04 AM" src="https://user-images.githubusercontent.com/13952758/79368879-f548b100-7f04-11ea-884f-990a13629eb5.png">
seems like mine is behaving differently, it wraps word on initially load then "adjusts" to the expect no wrap behavior once I resize the text area

Reply:
it could be a browser thing since I'm using Safari